data/localized/de_patch/text/conversations/companions/companion_cv_eder_intro.stringtable

```diff
@@ -236,7 +236,7 @@ Er betrachtet dich von Kopf bis Fuß und zieht eine Grimasse. „Glaube nicht, d
     </Entry>
     <Entry>
       <ID>44</ID>
-      <DefaultText>„Habe mich eingemischt.“</DefaultText>
+      <DefaultText>„Hat sich eingemischt.“</DefaultText>
```

Comment:
Den nehm ich mal hin, weil ich da jetzt nicht in den Kontext kucke.

Reply:
Antwort auf die Frage: "Warum wurde dein Vorarbeiter gehängt?"